Package updates

aylapear opened the pull request:
- Adding wqbench set of packages
- Updating readme
- Adding R-universe install steps #4 
- Fixing R CMD Check note #1

aylapear commented:
Converting to draft until https://github.com/poissonconsulting/ssdvignettes/pull/1 and https://github.com/poissonconsulting/envirotox/pull/3 are merged in

joethorley reviewed (changes requested):
ssdtests and ssdvignettes are no longer in DESCRIPTION - is this intentional?

aylapear commented:
Yes and no. The action is failing and I was trying to figure out why, it seems a few of the packages are struggling at being downloaded during the install step. It seems to be shinywqbench, wqbench, ssdtests, and ssdvignettes. I removed things one by one and was adding them back in to see what I could figure out. Everything is locally installing fine and I can't find any helpful error messages in the action so going to continue digging.

aylapear commented:
I'll remove you from reviewing until I get it more figured out, I thought it was ready but the actions were failing and then went down rabbit hole trying to fix it.

aylapear commented:
I got things working with the install now, the GitHub API is having some difficulties right now so waiting for them to all pass before requesting a review again. 

I think what was causing the issue is not the conflicting version numbers of ssdtools across the group but that some were using bcgov/ssdtools and others were using poissonconsulting/ssdtools@dev and pak can't resolve this as it doesn't know which one to download. 

I updated wqbench and shinywqbench to have a dev branch in the poissonconsulting repos that use the ssdtools@dev and changed this package to use ssdtools@dev. Since this package is installing the dev versions of things we may want to state that in the README and DESCRIPTION file because my initial assumption as a user is it would use the CRAN version and main versions.

joethorley commented:
Good work. I think ultimately we want to have the main versions use the main versions and then have a dev branch that uses the other dev branches.

aylapear commented:
Should we merge this PR in as is and then I can work on setting up the dev branch and main branch to be different in terms of their installation or are there additional changes you want to this PR?

joethorley commented:
Yes I think merge in as is
